### ISSUE TITLE
Knowledge Discovery through AI - How can explainable AI enhance your research?

### ISSUE THREAD

**Issue by lukaskln** [DKRZ-AIM/HAI-HI-unconference-2023#6]
# Knowledge Discovery through AI - How can explainable AI enhance your research?

# Description
Explore the Power of Data Interpretation at Helmholtz Imaging's Knowledge Discovery through Explainable AI (XAI) session!

Uncover the hidden treasures in your data with Helmholtz Imaging's engaging workshop focused on Deep Learning and XAI for Knowledge Discovery. Designed for both domain researchers and ML researchers, this session will provide an extensive understanding of the potential of XAI in deciphering complex data and transforming it into actionable insights.

Our session is set in two key phases:

1. **Presentation**: We commence with an in-depth introduction to XAI, focusing on its role in knowledge discovery. We will illustrate the power of XAI in making deep learning models comprehensible, thereby enabling the extraction of meaningful insights from data. This will be supplemented with real-life use-cases from the Health and Energy fields, where XAI has proven instrumental in shedding light on intricate data landscapes.

2. **Discussion**: Post-presentation, we transition into a dynamic discussion involving domain scientists. This interactive dialogue is designed to explore the possible applications of our XAI approach to their specific research questions. It will provide a platform for scientists to envision how XAI can be leveraged to unlock the potential of their data, enhancing their research outcomes.

Join us as we delve into the realm of XAI, fostering a rich exchange of ideas and catalyzing advancements in research. Unleash the power of your data with Helmholtz Imaging's Knowledge Discovery through XAI session.

# Organizational

## Organizer(s)
Lukas Klein [lukas.klein@dkfz.de](mailto:lukas.klein@dkfz.de)
Sebastian Ziegler [sebastian.ziegler@dkfz.de](mailto:sebastian.ziegler@dkfz.de) 

## Speakers
Lukas Klein [lukas.klein@dkfz.de](mailto:lukas.klein@dkfz.de)
Sebastian Ziegler [sebastian.ziegler@dkfz.de](mailto:sebastian.ziegler@dkfz.de) 

## Format
 Talks followed by open discussion

## Timeframe
1h

## Number of participants
min 5: max: 99

**Comment by SusanneWenzel** [comment]
@lukaskln I guess you would you need a screen, right? We have a few available, but possibly not for each session. Flipchart will be available

**Comment by lukaskln** [comment]
Yes, as we only have digital material we would definitely need a monitor/projector. Thanks for asking!

**Comment by SusanneWenzel** [comment]
thanks, noted

**Comment by SusanneWenzel** [comment]
@lukaskln if possible, please make a note here on the (rough) number of participants. Also don't forget to make a note here about the outcome of the session and, if applicable, future plans that came out of this session.
